indico/modules/events/management/controllers/settings.py

```diff
@@ -41,7 +46,21 @@ def _check_access(self):
         RHManageEventBase._check_access(self)  # mainly to trigger the legacy "event locked" check
 
     def _process(self):
-        return WPEventSettings.render_template('settings.html', self.event, 'settings')
+        # Check if any of the managers of the event already have a booking that overlaps with the event datetime
+        manager_ids = [p.user.id for p in self.event.acl_entries if p.user]
```

Comment:
that's everyone in the ACL, not just managers. the correct way to get only the managers is:

```python
manager_ids = {p.user.id
               for p in self.event.acl_entries
               if p.type == PrincipalType.user and p.has_management_permission()}
```